Platform/Ampere/buildfw.sh

```diff
@@ -0,0 +1,286 @@
+  echo "Could not find python3. Please install the python3 package."
+  exit 1
+fi
+
+OPTIONS=$(${GETOPT_COMMAND} -o t:b:fFhl:m:p: --long toolchain:,build:,linuxboot:,manufacturer:,platform:,flash,full-flash,help -- "$@")
+eval set -- "$OPTIONS"
+
+while true; do
+  case "$1" in
+    -t|--toolchain)
+      TOOLCHAIN=$2; shift 2;;
+    -b|--build)
+      BLDTYPE=$2; shift 2;;
+    -f|--flash)
+      FLASHFW=1; shift;;
+    -F|--full-flash)
+      FLASHFW=1; RESET_NV_STORAGE=1; shift;;
+    -l|--linuxboot)
+      LINUXBOOT=LinuxBoot; FLASHKERNEL=$2; shift 2;;
+    -m|--manufacturer)
+      MANUFACTURER=$2; shift 2;;
+    -p|--platform)
+      BOARD_NAME=$2;  shift 2;;
+    -h|--help)
+      usage; shift;;
+    --) shift; break;;
+    *) echo "Internal error ($1)!"; exit 1;;
+  esac
+done
+
+eval set -- ""
+
+BOARD_SETTINGS_CFG=edk2-platforms/Platform/${MANUFACTURER}/${BOARD_NAME}Pkg/${BOARD_NAME}BoardSetting.cfg
+OUTPUT_BIN_DIR=$PWD/Build/${BOARD_NAME}
+OUTPUT_BOARD_SETTINGS_BIN=${OUTPUT_BIN_DIR}/$(basename ${BOARD_SETTINGS_CFG}).bin
+
+export PACKAGES_PATH=$PWD:$PWD/edk2:$PWD/edk2-non-osi:$PWD/edk2-platforms:$PWD/edk2-platforms/Platform/${MANUFACTURER}/${BOARD_NAME}Pkg:$PWD/edk2-platforms/Features/Intel/Debugging:$PWD/edk2-platforms/Features:$PWD/edk2-platforms/Features/Intel
+
+case $(uname -m) in
+  "x86_64")
+    if [ "$TOOLCHAIN" = "GCC" -a -z "${GCC_AARCH64_PREFIX}" ]; then
+      echo "Error: need to define \$GCC_AARCH64_PREFIX since the native compiler won't work"
+      exit 1
+    fi
+    ;;
+esac
+
+if [ "${TOOLCHAIN}" = "CLANG" ]; then
+  TOOLCHAIN=CLANGDWARF
+fi
+
+if [ -n "${LINUXBOOT}" ]; then
+  if [ ! -f "${FLASHKERNEL}" ]; then
+    echo "LinuxBoot flashkernel file \"${FLASHKERNEL}\" does not exist."
+    exit 1
+  else
+    EXTRA_BUILD_FLAGS="-D LINUBOOT_FILE=${FLASHKERNEL}"
+  fi
+fi
+
+mkdir -p "${OUTPUT_BIN_DIR}"
+cp -v "${BOARD_SETTINGS_CFG}" "${OUTPUT_BIN_DIR}/$(basename ${BOARD_SETTINGS_CFG}).txt"
+python3 "${WORKSPACE}/edk2-ampere-tools/nvparam.py" -f "${BOARD_SETTINGS_CFG}" -o "${OUTPUT_BOARD_SETTINGS_BIN}"
+rm -fv "${OUTPUT_BOARD_SETTINGS_BIN}.padded"
+
+${MAKE_COMMAND} -C edk2/BaseTools -j ${BUILD_THREADS}
+
+. "${WORKSPACE}/edk2-platforms/Platform/Ampere/Tools/fw_ver.sh" UPDATE
+. edk2/edksetup.sh
+
+EDK2_SECURE_BOOT_ENABLE=${EDK2_SECURE_BOOT_ENABLE:-TRUE}
+EDK2_NETWORK_ENABLE=${EDK2_NETWORK_ENABLE:-TRUE}
+EDK2_INCLUDE_TFTP_COMMAND=${EDK2_INCLUDE_TFTP_COMMAND:-TRUE}
+EDK2_NETWORK_IP6_ENABLE=${EDK2_NETWORK_IP6_ENABLE:-TRUE}
+EDK2_NETWORK_ALLOW_HTTP_CONNECTIONS=${EDK2_NETWORK_ALLOW_HTTP_CONNECTIONS:-TRUE}
+EDK2_NETWORK_TLS_ENABLE=${EDK2_NETWORK_TLS_ENABLE:-TRUE}
+EDK2_REDFISH_ENABLE=${EDK2_REDFISH_ENABLE:-TRUE}
+EDK2_PERFORMANCE_MEASUREMENT_ENABLE=${EDK2_PERFORMANCE_MEASUREMENT_ENABLE:-TRUE}
+EDK2_TPM2_ENABLE=${EDK2_TPM2_ENABLE:-TRUE}
+
+if [ "${BLDTYPE}" = "RELEASE" ]; then
+  EDK2_HEAP_GUARD_ENABLE=FALSE
+else
+  EDK2_HEAP_GUARD_ENABLE=TRUE
+fi
+
+UPD720202_ROM_FILE="K2026090.mem"
+EXTRA_BUILD_FLAGS=""
```

Comment:
This resets the flag of LinuxBoot at Line 162 `EXTRA_BUILD_FLAGS="-D LINUBOOT_FILE=${FLASHKERNEL}"`